modules/ROOT/pages/managing-connections.adoc

```diff
@@ -49,6 +49,21 @@ var bucket = cluster.openBucket('protectedBucket', function(err) {
 });
 ----
 
+[NOTE]
+====
+
+From Couchbase Node.js SDK 2.5.0 (using _libcouchbase_ 2.9.2), [.api]`AlternateAddress` is implemented, for connecting to nodes in a NATed environment, such as Docker containers using portmapping.
```

Comment:
You use 'AlternateAddress' here, but then call it Multi-Network Configuration below.  Why is this?

Reply:
I'm going with AlternateAddress as the implementation of multi network configuration, 
we did something similar in the RTO / Tracing docs - what would be a better alternative?